### PR TITLE
feat: add google analytics

### PR DESCRIPTION
**Pull request by tinashechiraya**

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 2 min 25 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 2 min 23 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 3 min 32 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 2 min 25 sec

**Review by NyakudyaA — APPROVED** [review]

File: .example.env [hunk 1]
```diff
@@ -66,3 +66,6 @@ DJANGO_SUPERUSER_PASSWORD=Gs10w29k8*&
 
 #Tensorflow memmory limit
 TF_MEMORY_LIMIT_GB=0.5
+
+# Google analytics env var
+GOOGLE_ANALYTICS_TRACKING_CODE=
```
Comment (NyakudyaA) [inline .example.env:71]: @tharanathkartoza The env should only be set for production env

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 12 min

**Review by NyakudyaA — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 46 sec

**Pull request merged**